win32/build/build.ninja

```diff
@@ -0,0 +1,115 @@
+#  +----------------------------------------------------------------------+
+#  | Copyright (c) The PHP Group                                          |
+#  +----------------------------------------------------------------------+
+#  | This source file is subject to version 3.01 of the PHP license,      |
+#  | that is bundled with this package in the file LICENSE, and is        |
+#  | available through the world-wide-web at the following url:           |
+#  | https://www.php.net/license/3_01.txt                                 |
+#  | If you did not receive a copy of the PHP license and are unable to   |
+#  | obtain it through the world-wide-web, please send a note to          |
+#  | license@php.net so we can mail you a copy immediately.               |
+#  +----------------------------------------------------------------------+
+#  | Author: Wez Furlong <wez@thebrainroom.com>                           |
+#  +----------------------------------------------------------------------+
+#
+# This is the makefile template for the win32 build
+
+CC="${PHP_CL}"
+LD="${LINK}"
+MC="${MC}"
+MT="${MT}"
+RE2C="${RE2C}"
+PGOMGR="${PGOMGR}"
+PHP_BUILD=${PHP_BUILD}
+
+msvc_deps_prefix = Hinweis: Einlesen der Datei:
```

Comment:
For an English VS, just remove that line. For other languages, see https://ninja-build.org/manual.html#_deps.